spec/factories/submission.rb

```diff
@@ -2,9 +2,13 @@
   factory :submission do
     description { Faker::Lorem.paragraph }
     state { :inactive }
-    deadline { Date.tomorrow }
 
     association :fund, factory: :fund_activity
     association :organisation
+
+    trait :active do
+      state { :active }
+      deadline { Date.tomorrow }
```

Comment:
Again, I feel like this being a little furhter might reflect reality a little closer, I am stuggling to reason why though!?